cfgmgr/vlanmgr.cpp

```diff
@@ -122,6 +123,94 @@ bool VlanMgr::removeHostVlan(int vlan_id)
     return true;
 }
 
+
+bool VlanMgr::updateVlanStatusInit(vector<FieldValueTuple> &fvVector, const int vlan_id)
+{
+    fvVector.push_back(make_pair("admin_status", "up"));
+    fvVector.push_back(make_pair("member_status", "down"));
+    fvVector.push_back(make_pair("real_status", "down"));
```

Comment:
why not "oper_status" instead of "real_status"

Reply:
thanks，I'll change "real_status" to "oper_status"